share/genbuild.sh

```diff
@@ -18,17 +18,14 @@ else
     exit 1
 fi
 
-GIT_TAG=""
-GIT_COMMIT=""
+GIT_TAG="${GIT_TAG:-}"
+GIT_COMMIT="${GIT_COMMIT:-}"
```

Comment:
6b6b8e7edeba3360738fb40ebc12872a81089967

Should we check `GIT_TAG` and `GIT_COMMIT` for emptiness, and run the following `if` block when they are empty?

Reply:
No, I did it this way specifically to allow for `GIT_TAG` and `GIT_COMMIT` to be fallbacks. So if we are in the git work tree, then it won't be possible to accidentally override them with the incorrect values.